spinn_machine/machine.py

```diff
@@ -806,19 +806,7 @@ def total_cores(self):
         return sum(
             1 for chip in self.chips for _processor in chip.processors)
 
-    def remove_unreachable_chips(self):
-        """ Remove chips that can't be reached or that can't reach other chips\
-            due to missing links
-        """
-        for xy in self._unreachable_incoming_chips:
-            if xy in self._chips:
-                del self._chips[xy]
-        for xy in self._unreachable_outgoing_chips:
-            if xy in self._chips:
-                del self._chips[xy]
-
-    @property
-    def _unreachable_outgoing_chips(self):
+    def unreachable_outgoing_chips(self):
```

Comment:
I wonder whether this (and `unreachable_incoming_chips`) ought to be done as generators rather than iterators.